ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController.swift

```diff
@@ -887,6 +882,13 @@ public class BrowserViewController: UIViewController {
     // Adding Screenshot Service Delegate to browser to fetch full screen webview screenshots
     currentScene?.screenshotService?.delegate = self
 
+    // Add Regular tabs to Sync Chain
+    if Preferences.Chromium.syncOpenTabsEnabled.value {
+      executeAfterSetup {
```

Comment:
Check this pref in the closure in case the pref changes for some reason between setup